pkg/storer/internal/stampindex/stampindex.go

```diff
@@ -98,17 +109,17 @@ func (i *Item) Unmarshal(bytes []byte) error {
 		return errStampItemUnmarshalInvalidSize
 	}
 	nsLen := int(binary.LittleEndian.Uint64(bytes))
-	if len(bytes) != 8+nsLen+swarm.HashSize+swarm.StampIndexSize+swarm.StampTimestampSize+swarm.HashSize+1 {
+	if len(bytes) != 8+nsLen+swarm.HashSize+swarm.StampIndexSize+swarm.StampTimestampSize+swarm.HashSize+swarm.HashSize+1 {
```

Comment:
Is the same size as above? If yes , can we add this into a constant ?

Reply:
Not necessarily and it cannot be constant because it depends on the namespace (variable length)